backend/init/init.go

```diff
@@ -0,0 +1,69 @@
+// Package init contains the list of backends that can be initialized and
+// basic helper functions for initializing those backends.
+package init
+
+import (
+	"sync"
+
+	"github.com/hashicorp/terraform/backend"
+
+	backendlegacy "github.com/hashicorp/terraform/backend/legacy"
+	backendlocal "github.com/hashicorp/terraform/backend/local"
+	backendconsul "github.com/hashicorp/terraform/backend/remote-state/consul"
+	backendinmem "github.com/hashicorp/terraform/backend/remote-state/inmem"
```

Comment:
I don't think so since its no different than our `internal_plugin_list.go` file. That file happens to be generated now (it used to not!), but its just like this. The full list of built-in things has to be enumerated _somewhere_ and `backend/init` is where that's happening. I think as long as we do this only in one place we're good to go, cause its going to happen somewhere anyways.